Documentation/upgrading.md

```diff
@@ -1,173 +1,149 @@
+# Upgrading self-hosted Kubernetes
 
-# Upgrading Self-hosted Kubernetes
+"Self-hosted" Kubernetes clusters run the apiserver, scheduler, controller-manager, kube-dns, kube-proxy, and flannel or calico as pods, like ordinary applications (except with taint tolerations). This allows upgrades to be performed in-place using (mostly) `kubectl` as an alternative to re-provisioning.
```

Comment:
nit: maybe drop "(except with taint tolerations)" -- I would still consider that part of an ordinary application, just another scheduling mechanic.

Reply:
Oh just saw we had similar wording in the deleted section. Still think it's extraneous :)

Reply:
agreed, its not very important for the person getting a high level summary